arch/src/main/java/dk/ufst/arch/Store.kt

```diff
@@ -8,21 +8,27 @@ interface LocalStore<Value, Action> {
     fun send(action: Action)
 }
 
+interface GlobalStore<Value, Action, Environment> {
+    fun sendAction(action: Action)
+    fun subscribe(subscriber: (Value) -> Unit)
+    fun desubscribe(subscriber: (Value) -> Unit)
+    val subscriberCount: Int
+    val value: Value
+}
+
 @Suppress("unused", "MemberVisibilityCanBePrivate")
```

Comment:
Not super important, but it looks like we can remove `MemberVisibilityCanBePrivate`?